Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/go-toolset:1.19 as builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17 AS builder
```

Comment:
No, we need to use only public images for the konflux integration

Reply:
(I just changed this in https://github.com/openshift/image-based-install-operator/pull/70)

Reply:
This one https://github.com/openshift/image-based-install-operator/pull/77/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R19

Reply:
I don't see registry.access.redhat.com/ubi8/go-toolset:1.22

Reply:
The reason we need >= 1.22 we need `DataImage` from github.com/metal3-io/baremetal-operator/apis/metal3.io/v1alpha1 that requires go >= 1.22
`go: github.com/openshift/baremetal-operator/apis@v0.0.0-20240611081634-a92cc4b9ae8b requires go >= 1.22; switching to go1.22.6`